xoinvader/animation.py

```diff
@@ -91,14 +91,23 @@ def update(self):
 class Animation(object):
```

Comment:
>Animation.update method splitted

I really think that present time and imperative voice should be used in descibing changes not only in commit message caption. So if you're not referring to the state file was in prior to your changes, you shouldn't use past tense. Thus the correct phrase IMO should be

>Split Animation.update method in two separate update methods ...

And so on.